[Bug][Docker] error while building tubemq image

### What happened

An error occurred when build tubemq image.

The command is: 

```shell
mvn --batch-mode --update-snapshots -e -V clean package -DskipTests -Pdocker
```

The working directory is the root directory of inlong.

See https://github.com/shink/incubator-inlong/runs/4671539072?check_suite_focus=true

```java
Caused by: com.spotify.docker.client.exceptions.DockerException: The command '/bin/sh -c wget -O "/tmp/protobuf-${PROTOBUF_VERSION}.zip" "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip"     && unzip "/tmp/protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/"     && cd /opt/protobuf-${PROTOBUF_VERSION}     && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install     && rm -f "/tmp/protobuf-${PROTOBUF_VERSION}.zip"' returned a non-zero code: 8
    at com.spotify.plugin.dockerfile.LoggingProgressHandler.handleError (LoggingProgressHandler.java:105)
    at com.spotify.plugin.dockerfile.LoggingProgressHandler.progress (LoggingProgressHandler.java:63)
    at com.spotify.docker.client.DefaultDockerClient$BuildProgressHandler.progress (DefaultDockerClient.java:312)
    at com.spotify.docker.client.ProgressStream.tail (ProgressStream.java:77)
    at com.spotify.docker.client.DefaultDockerClient$ResponseTailReader.call (DefaultDockerClient.java:2754)
    at com.spotify.docker.client.DefaultDockerClient$ResponseTailReader.call (DefaultDockerClient.java:2738)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:624)
    at java.lang.Thread.run (Thread.java:748)
```

### What you expected to happen

All images can be built correctly.

### How to reproduce

Enter the root directory of inlong, execute the following command.

```shell
mvn --batch-mode --update-snapshots -e -V clean package -DskipTests -Pdocker
```

### Environment

- ubuntu: 20.04
- docker: 20.10.12
- java: 1.8.0_261
- maven: 3.8.4


### InLong version

master

### InLong Component

InLong TubeMQ

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile
+++ b/inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile
@@ -21,7 +21,7 @@ FROM gcc:5.4.0 as protobuf
 RUN apt-get update && apt-get install -y unzip \
     && rm -rf /var/lib/apt/lists/
 ARG PROTOBUF_VERSION=2.5.0
-RUN curl -LO "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
+RUN curl -LOk "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
     && unzip "protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
     && cd /opt/protobuf-${PROTOBUF_VERSION} \
     && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install \

--- a/inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile
+++ b/inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile
@@ -21,8 +21,8 @@ FROM gcc:5.4.0 as protobuf
 RUN apt-get update && apt-get install -y unzip \
     && rm -rf /var/lib/apt/lists/
 ARG PROTOBUF_VERSION=2.5.0
-RUN wget -O "/tmp/protobuf-${PROTOBUF_VERSION}.zip" "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
-    && unzip "/tmp/protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
+RUN curl -LO "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
+    && unzip "protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
     && cd /opt/protobuf-${PROTOBUF_VERSION} \
     && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install \
     && rm -f "/tmp/protobuf-${PROTOBUF_VERSION}.zip"

--- a/inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile
+++ b/inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile
@@ -25,7 +25,7 @@ RUN curl -LOk "https://github.com/protocolbuffers/protobuf/releases/download/v${
     && unzip "protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
     && cd /opt/protobuf-${PROTOBUF_VERSION} \
     && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install \
-    && rm -f "/tmp/protobuf-${PROTOBUF_VERSION}.zip"
+    && rm -f "protobuf-${PROTOBUF_VERSION}.zip"
 # for building tubemq
 FROM maven:3-openjdk-8
 WORKDIR /tubemq/

--- a/inlong-tubemq/tubemq-docker/tubemq-cpp/Dockerfile
+++ b/inlong-tubemq/tubemq-docker/tubemq-cpp/Dockerfile
@@ -22,7 +22,7 @@ RUN wget https://www.openssl.org/source/openssl-1.1.0f.tar.gz --no-check-certifi
     && tar -xzvf openssl-1.1.0f.tar.gz \
     && rm openssl-1.1.0f.tar.gz && cd openssl-1.1.0f \
     && ./config && make && make install
-RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/protobuf-cpp-3.13.0.tar.gz \
+RUN curl -LOk https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/protobuf-cpp-3.13.0.tar.gz \
     && tar -xzvf protobuf-cpp-3.13.0.tar.gz && rm protobuf-cpp-3.13.0.tar.gz \
     && cd protobuf-3.13.0 && ./autogen.sh \
     && ./configure CXXFLAGS=-fPIC && make \

--- a/inlong-tubemq/tubemq-docker/tubemq-cpp/Dockerfile
+++ b/inlong-tubemq/tubemq-docker/tubemq-cpp/Dockerfile
@@ -18,7 +18,7 @@
 #
 FROM rikorose/gcc-cmake:gcc-4
 RUN apt-get remove openssl -y
-RUN wget https://www.openssl.org/source/openssl-1.1.0f.tar.gz --no-check-certificate \
+RUN curl -LOk https://www.openssl.org/source/openssl-1.1.0f.tar.gz \
     && tar -xzvf openssl-1.1.0f.tar.gz \
     && rm openssl-1.1.0f.tar.gz && cd openssl-1.1.0f \
     && ./config && make && make install

